pyproject.toml

```diff
@@ -27,66 +27,66 @@ etl-metadata-export = 'etl.metadata_export:cli'
 python = "^3.9"
 owid-catalog = {path = "vendor/owid-catalog-py", develop = true}
 walden = {path = "vendor/walden", develop = true}
-click = "^8.0.1"
-PyYAML = "^5.4.1"
-papermill = "^2.3.3"
-jupyterlab = "^3.1.13"
-mysqlclient = "^2.0.3"
-SPARQLWrapper = "^1.8.5"
-openpyxl = "^3.0.9"
-sh = "^1.14.2"
-Unidecode = "^1.3.2"
-python-dotenv = "^0.19.0"
+click = ">=8.0.1"
+PyYAML = ">=5.4.1"
+papermill = ">=2.3.3"
+jupyterlab = ">=3.1.13"
+mysqlclient = ">=2.0.3"
+SPARQLWrapper = ">=1.8.5"
+openpyxl = ">=3.0.9"
+sh = ">=1.14.2"
+Unidecode = ">=1.3.2"
+python-dotenv = ">=0.19.0"
 # NOTE: the version is currently fixed, but we should update as soon as https://github.com/frictionlessdata/frictionless-py/issues/1143
 #   gets resolved
 frictionless = {version = "4.38.0", extras = ["pandas"]}
-thefuzz = "^0.19.0"
-python-Levenshtein = "^0.12.2"
-regex = "^2022.1.18"
-wikipedia = "^1.4.0"
-pandas = "^1.4.0"
-numpy = "^1.22.1"
-pydantic = "^1.9.0"
-structlog = "^21.5.0"
-sqlmodel = "^0.0.6"
-rich = "^12.1.0"
-owid-datautils = {git = "https://github.com/owid/owid-datautils-py.git", tag = "v0.5.1-alpha"}
-fuzzywuzzy = "^0.18.0"
-rich-click = "^1.5.1"
-tenacity = "^8.0.1"
-simplejson = "^3.17.6"
-bugsnag = "^4.2.1"
-xlrd = "^2.0.1"
-PyPDF2 = "^2.11.1"
-dvc = {extras = ["s3"], version = "^2.38.1"}
-"ruamel.yaml" = "^0.17.21"
-owid-repack = "^0.1.1"
+thefuzz = ">=0.19.0"
+python-Levenshtein = ">=0.12.2"
+regex = ">=2022.1.18"
+wikipedia = ">=1.4.0"
+pandas = ">=1.4.0"
+numpy = ">=1.22.1"
```

Comment:
I don't have a strong opinion on other libraries, but I think I feel more comfortable having pandas constrained. We were even considering pinning it to a particular version. I think it's one that we can manually update every now and then, and be aware of changes.

Reply:
Yeah, we've agreed on this a while back and there's a plan to pin it to our current version (which is not 1.4.0, but 1.5.2 I think). We can either do it in this PR or I can do it later separately.

Reply:
My thinking has shifted a little on reflection.

So far, I don't think we have evidence that they've broken more than they've fixed with new releases.

Our only known problem arose because we began using different Pandas features, not because we upgraded unexpectedly.

Unless we do find evidence that they're doing a bad job upstream, I actually think we should be more bullish about tracking recent Pandas releases.

The 1.5.1 to 1.5.2 comparison has dragged out, since there was tons of spurious changes just because our ETL dataset had changed between the two runs, so now I'm having to run them both from scratch again to compare, which takes a while. I'll see how it goes and maybe update my perspective if we find something there.

Reply:
I still think it's better to do it in a controlled way by opening special PR and checking some kind of automatic validation [like datadiff](https://github.com/owid/etl/pull/694). They quite often deprecate features or add warnings with new versions (like the most recent [flood of warnings](https://github.com/owid/etl/pull/700)), so it's better to check beforehand. (I got also bitten by pandas upgrades in the past. They're doing better job for sure, but I'd still be careful given that everything we have depends on pandas).

> The 1.5.1 to 1.5.2 comparison has dragged out

(By the way we're already on 1.5.2, so it might be better to do 1.5.2 vs 1.5.3 which should be out soon)

Reply:
@larsyencken what's your opinion on pinning it to 1.5.2? We recently had [an issue](https://github.com/owid/etl/issues/733) caused by update from 1.5.1 -> 1.5.2 so I'd rather pin it to 1.5.2 and update it in a controlled fashion.